### PR TITLE
Update Available Models

### PR DESCRIPTION
**Pull request by abdulfatir**
*Issue #, if available:*

*Description of changes:* 
Updating the list of available models in GluonTS docs.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

**Comment by abdulfatir** [comment]
@lostella what should I write under "Architecture/method" for DeepNPTS. Also, is the MQF2 model functional?

**Comment by lostella** [comment]
@abdulfatir MLP or feed-forward network is fine. Torch estimator classes that are tested [here](https://github.com/awslabs/gluonts/blob/dev/test/torch/model/test_estimators.py) can be assumed to be functional

File: docs/getting_started/models.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ Temporal Fusion Transformer<br>[Lim et al. 2021][Lim2021]    | Global       | Un
 Transformer<br>[Vaswani et al. 2017][Vaswani2017]            | Global       | Univariate               | MLP, multi-head attention | [MXNet][Transformer]
 WaveNet<br>[van den Oord et al. 2016][vanDenOord2016]        | Global       | Univariate               | Dilated convolution | [MXNet][WaveNet]
 SimpleFeedForward                                            | Global       | Univariate               | MLP | [MXNet][SFF_mx], [PyTorch][SFF_torch]
+DeepNPTS                                                     | Global       | Univariate               | MLP | [PyTorch][DeepNPTS_torch]
+MQF2<br>[Kan et al. 2022][Kan2022]                           | Global       | Univariate               | RNN, ICNN | [PyTorch][MQF2_torch]
```
Comment (abdulfatir) [inline docs/getting_started/models.md:19]: @lostella do you know if "Univariate" is correct for MQF2? Looks like this model is "multivariate" in time but actually univariate in target dim. 

cc @benidis

Comment (lostella) [reply docs/getting_started/models.md:19]: yes, this is correct

**Review by lostella — APPROVED** [review]

**Pull request merged**